policyengine_us/variables/gov/states/nc/ncdhhs/tanf/income/nc_tanf_countable_earned_income.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class nc_tanf_countable_earned_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "North Carolina TANF countable earned income"
+    unit = USD
+    definition_period = YEAR
```

Comment:
```suggestion
    definition_period = YEAR
    defined_for = StateCode.NC 
```

Reply:
can we add this to all vars that are not defined fro an eligibility variable, just to minimize the computation for other states